nodes/src/nodes/ml_sklearn/code.py

```
@@ -0,0 +1,33 @@
+import pickle
+import os
+
+class PreProcessor:
+    '''ML Sklearn Prediction Node'''
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)